src/test/java/org/opendatakit/aggregate/format/table/CsvFormatterWithFiltersTest.java

```diff
@@ -0,0 +1,75 @@
+package org.opendatakit.aggregate.format.table;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyBoolean;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.opendatakit.aggregate.constants.common.FormElementNamespace;
+import org.opendatakit.aggregate.datamodel.FormElementModel;
+import org.opendatakit.aggregate.form.IForm;
+import org.opendatakit.aggregate.format.Row;
+import org.opendatakit.aggregate.format.element.ElementFormatter;
+import org.opendatakit.aggregate.submission.Submission;
+import org.opendatakit.common.persistence.exception.ODKDatastoreException;
+import org.opendatakit.common.web.CallingContext;
+
+public class CsvFormatterWithFiltersTest {
+
+  private IForm form = mock(IForm.class);
+  private StringWriter actual;
+  private PrintWriter pr;
+  private Submission s;
+  private Row row;
+  private ArrayList<String> testList = new ArrayList<String>();
+  private ArrayList<Submission> subs = new ArrayList<Submission>();
```

Comment:
Minor issue: no need for specifying the type on the `ArrayList`. We can use `<>` instead.